code/game/machinery/door_control.dm

```diff
@@ -1,6 +1,5 @@
 #define CONTROL_POD_DOORS 0
 #define CONTROL_NORMAL_DOORS 1
-#define CONTROL_EMITTERS 2
 #define CONTROL_DROPSHIP 3
```

Comment:
```suggestion
#define CONTROL_DROPSHIP 2
```

Reply:
I dont know if that could cause conflicts elsewhere in the code.

Reply:
Anyhow I just realised with the shuttle port we can even remove the need for this system as the TG one can call close on all the airlocks a different way on TG.

Reply:
This is being used in two `switch` checks. The change won't break anything.

Reply:
Okay. I will sort it tonight.